config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml

```diff
@@ -163,6 +164,13 @@ presubmits:
             make e2e-bootstrap e2e-helm-deploy e2e-vault
         securityContext:
           privileged: true
+        resources:
+          requests:
+            cpu: "4"
+            memory: "8Gi"
```

Comment:
I think I allocated more memory than required, what value should I use, maybe 4 or 6 Gi?

Reply:
6 Gi is fine.

Reply:
Done!